include/dlaf/matrix/internal/tile_future_manager.h

```diff
@@ -8,6 +8,8 @@
 // SPDX-License-Identifier: BSD-3-Clause
 //
 
+#pragma once
```

Comment:
Again unrelated to the actual changes here, but we're thinking about making our "inspect" tool usable by others as well: https://github.com/pika-org/pika/issues/6. One of the checks it does is make sure all header files have `#pragma once`. It's definitely low on our priority list, but if you think it'd be useful we could try to do that sooner rather than later.

Alternatively, a low-tech solution is to just put something simple with grep into CI. Our inspect tool is in principle also nothing more than a glorified grep.

Reply:
will be addressed in #461.